logcall/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
I mean that we might want to improve the file name following Odoo recommendations: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules
Starting with the standard base odoo module hence my suggestion (crm_logcall). Nevertheless later I saw there is a crm_voip_logcall and now I am confused about the separation between both.